lib/service_check.pm

```diff
@@ -18,6 +18,13 @@ use utils;
 use base 'opensusebasetest';
 use strict;
 use warnings;
+use File::Basename;
+
+# load tests/console path in @INC
+BEGIN {
+    unshift @INC, dirname(__FILE__) . '/../tests/console';
+}
+use http_srv;
```

Comment:
can you move this line to the top  so use section will be in one place?

Reply:
done,thanks

Reply:
moved to top and it complains with ""lib/service_check.pm: Code before strictures are enabled at line 16, column 1.  See page 429 of PBP.  (Severity: 5)